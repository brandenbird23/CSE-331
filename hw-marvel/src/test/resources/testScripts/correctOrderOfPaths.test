# Tests to find if paths are printed in lexicographical order
# Testing when there are multiple paths

LoadGraph nfl nflPlayers.csv

FindPath nfl TOM-BRADY ROB-GRONKOWSKI