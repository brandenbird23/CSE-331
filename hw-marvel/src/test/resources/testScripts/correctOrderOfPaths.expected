# Tests to find if paths are printed in lexicographical order
# Testing when there are multiple paths

loaded graph nfl

path from TOM-BRADY to ROB-GRONKOWSKI:
ROB-GRONKOWSKI to TOM-BRADY via BUCS